src/strategies/total-axion/index.ts

```diff
@@ -0,0 +1,28 @@
+import fetch from 'cross-fetch';
+
+export const author = 'Jordan Travaux';
+export const version = '0.1.0';
+
+const getAddressTotalAxion = async (address: string) => {
+  const rawResponse = await fetch(`https://gateway.axntoday.com/v1/staker/${address}`);
```

Comment:
One thing i see missing is the "snapshot", if you don't have a way to get staker amount at a specific block number the results of the proposals will change in live and never be fixed, this can be problematic.

Reply:
Ah, good point. Will implement a fix!